src/components/ShellView/ResultGraph.vue

```diff
@@ -341,6 +360,30 @@ export default {
       this.graphCreated = true;
     },
 
+    hideNode() {
+      const currentSelectedNode = this.g6graph.findAllByState('node', 'click')[0];
+      const nodeId = currentSelectedNode.getModel().id;
+      this.hiddenNodes.push(currentSelectedNode);
+      currentSelectedNode.hide();
+
+      const relatedEdges = this.g6graph.getEdges().filter((edge) => {
+        const edgeModel = edge.getModel();
+        return edgeModel.source === nodeId || edgeModel.target === nodeId;
+      });
+
+      relatedEdges.forEach((edge) => {
+        this.hiddenRels.push(edge);
+        edge.hide();
+      });
```

Comment:
I think you need to deselect the node after is hidden. Otherwise the right panel is still showing the hidden node's info.